config/channel/distributed/README.md

```diff
@@ -82,17 +82,16 @@ Example value for a multi-broker Kafka (must be base64 encoded):
 ```
 
 The associated auth information is specified in a Kubernetes Secret in the
-`knative-eventing` namespace which has been labelled as
-`eventing-kafka.knative.dev/kafka-secret="true"`. For the `kafka` and `custom`
-Admin Types (see above) there should be exactly 1 such Secret. For the `azure`
-Admin Type (see above) multiple such Secrets are possible, each representing a
-different EventHub Namespace. In that case Topics will be load balanced across
-all EventHub Namespaces. The [kakfa-secret.yaml](300-kafka-secret.yaml) is
-included in the config directory, but must be modified to hold real values. The
-values from this file will override the username/password in the
-[eventing-kafka-configmap.yaml](300-eventing-kafka-configmap.yaml). It is also
-expected that the `Config.Net.SASL` and `Config.Net.TLS` are enabled to perform
-the required authentication with the Kafka cluster.
+`knative-eventing` namespace which is by default named `kafka-cluster` (the
+kafka.authSecretName and/or kafka.authSecretNamespace fields in the
+[eventing-kafka-configmap.yaml](300-eventing-kafka-configmap.yaml) may be
+changed if another secret location is desired). The
+[kakfa-secret.yaml](300-kafka-secret.yaml) is included in the config directory,
+but must be modified to hold real values. The values from this file will
+override the username/password in the
+[eventing-kafka-configmap.yaml](300-eventing-kafka-configmap.yaml).
+It is also expected that the `Config.Net.SASL` and `Config.Net.TLS` are enabled
```

Comment:
can you be more specific and say where in the configmap the username/password are (the sarama yaml string with mention of object path).  My first thought was "where do have that in the configmap" and then remembered it was in the sarama yaml ; )

Reply:
Sure; added references to Net.SASL.User and Net.SASL.Password explicitly